econml/dml.py

```diff
@@ -333,55 +435,97 @@ class DMLCateEstimator(_DMLCateEstimatorBase):
         by `np.random`.
 
-    inference: string, inference method, or None
-        Method for performing inference.  This estimator supports 'bootstrap'
-        (or an instance of `BootstrapOptions`)
     """
 
     def __init__(self,
-                 model_y, model_t, model_final=LinearRegression(fit_intercept=False),
+                 model_y=LassoCV(), model_t=LassoCV(),
                  featurizer=PolynomialFeatures(degree=1, include_bias=True),
+                 linear_first_stages=True,
                  discrete_treatment=False,
                  n_splits=2,
-                 random_state=None,
-                 inference=None):
+                 random_state=None):
         super().__init__(model_y=model_y,
                          model_t=model_t,
-                         model_final=model_final,
+                         model_final=StatsModelsWrapper(),
                          featurizer=featurizer,
-                         sparseLinear=False,
+                         linear_first_stages=linear_first_stages,
                          discrete_treatment=discrete_treatment,
                          n_splits=n_splits,
-                         random_state=random_state,
-                         inference=inference)
+                         random_state=random_state)
+
+        self.statsmodelswrapper = self._model_final._model
 
+    def _get_inference_options(self):
+        # add statsmodels to parent's options
+        options = super()._get_inference_options()
+        options.update(statsmodels=StatsModelsInference)
+        return options
 
-class SparseLinearDMLCateEstimator(_DMLCateEstimatorBase):
+    # override only so that we can update the docstring to indicate support for `StatsModelsInference`
+    def fit(self, Y, T, X=None, W=None, sample_weight=None, inference=None):
+        """
+        Estimate the counterfactual model from data, i.e. estimates functions τ(·,·,·), ∂τ(·,·).
+
+        Parameters
+        ----------
+        Y: (n × d_y) matrix or vector of length n
+            Outcomes for each sample
+        T: (n × dₜ) matrix or vector of length n
+            Treatments for each sample
+        X: optional (n × dₓ) matrix
+            Features for each sample
+        W: optional (n × d_w) matrix
+            Controls for each sample
+        sample_weight: optional (n,) vector
+            Weights for each row
+        inference: string, `Inference` instance, or None
+            Method for performing inference.  This estimator supports 'bootstrap'
+            (or an instance of `BootstrapInference`) and 'statsmodels' (or an instance of 'StatsModelsInference`)
+
+        Returns
+        -------
+        self
+        """
+        return super().fit(Y, T, X=X, W=W, sample_weight=sample_weight, inference=inference)
+
+    @property
+    def effect_op(self):
+        return self._model_final.effect_op
+
+    @property
+    def coef_(self):
```

Comment:
similar comment here about coef_ (I'm not even sure why we re-implemented coef_ here and not left the super().coef_